frontend/src/queries/query.ts

```diff
@@ -87,7 +87,7 @@ export async function pollForResults(
 
     while (performance.now() - pollStart < QUERY_ASYNC_TOTAL_POLL_SECONDS * 1000) {
         await delay(currentDelay, methodOptions?.signal)
-        currentDelay = Math.min(currentDelay * 2, QUERY_ASYNC_MAX_INTERVAL_SECONDS * 1000)
+        currentDelay = Math.min(currentDelay * 1.25, QUERY_ASYNC_MAX_INTERVAL_SECONDS * 1000)
```

Comment:
I'm +1 on shortening intervals, but I think your numbers are off 🤔 

We start with `currentDelay = 300` right away and there is a max of 5 sec.
So it's more like
(issue query)
0.30s wait and check
0.60 ...
1.20
2.40
4.80
5.00
....
That's not that bad.

Should we reduce the 5 sec maximum?

Reply:
I like your suggestion, let's reduce the total to 3 seconds.

In terms of the intervals, I think we're talking about the same thing but the numbers I posted are cumulative total query time. 

```
Your numbers (my numbers)
0.30s (.3s)
0.60s (.9s)
1.20s (2.1s)
2.40s (4.5s)
4.80s (9.3s)
5.00s (14.3s)
```